eclass/qt5-build.eclass

```diff
@@ -859,4 +898,61 @@ qt5_regenerate_global_qconfigs() {
 	else
 		ewarn "${qconfig_pri} or ${qconfig_pri_orig} does not exist or is not a regular file"
 	fi
+
+	if [[ ${QT5_MINOR_VERSION} -ge 9 ]]; then
```

Comment:
minor thing: split this whole new piece of code into a separate function? then you can call it only in 5.9 and later, which reduces the levels of indentation inside the new function.